entrypoint.sh

```diff
@@ -2,6 +2,14 @@
 
 #default values
 export CHAIN_ID=${CHAIN_ID-0x2323}
+#mkdir if not exists
+mkdir -p parity_data
+#if parity_data is empty, use parity_data.default as initial contents
```

Comment:
assuming I guess that it exists? That default thing is created after first successful run?